index.js

```diff
@@ -920,6 +920,20 @@ const fileType = input => {
 		};
 	}
 
+	if (checkString('Creative Voice File')) {
+		return {
+			ext: 'voc',
+			mime: 'audio/x-voc'
+		};
+	}
+
+	if (check([0x0B, 0x77])) {
+		return {
+			ext: 'ac3',
+			mime: 'audio/vnd.dolby.dd-raw'
```

Comment:
Why not `audio/ac3`? [ref](https://www.iana.org/assignments/media-types/media-types.xhtml)

Reply:
Yeah, copy-pasted from `file` output, should have checked.

Reply:
Hmmm. I guess it's actually incorrect. https://tools.ietf.org/html/rfc4184 is onlyRTP.

Reply:
I don't think it technically required to exclusively bound to `audio/ac3` to RTP, since a file can never be RTP. But you are right, it looks like it is not stated anywhere audio/ac3 should be used for a stream of file according the ATSC standard. I see quite some source code using `audio/vnd.dolby.dd-raw` (but also some uses  `audio/ac3`). I concur to keep the PR as is and recommend to merge it.